libs/langchain/langchain/chat_models/base.py

```diff
@@ -55,19 +68,35 @@ def init_chat_model(
                 - gemini... -> google_vertexai
                 - command... -> cohere
                 - accounts/fireworks... -> fireworks
+        configurable: Whether model parameters are configurable. Defaults to True if
+            model is None or config_prefix is non-null or configure_any is True.
+        config_prefix: If config_prefix is a non-empty string then model will be
+            configurable at runtime via the
+            ``config["configurable"]["{config_prefix}_model"]`` and
+            ``config["configurable"]["{config_prefix}_model_provider"]`` keys. If
+            config_prefix is an empty string then model will be configurable via
+            ``config["configurable"]["model"]`` and
+            ``config["configurable"]["model_provider"]`` keys.
+        configure_any: If True then any parameter can be passed to model initializer as
```

Comment:
In what situations would we want `configure_any` to be False?

Reply:
if dev doesn't want to expose certain params, or there's name collisions between model params and params for something else (eg base_url)

Reply:
Seems like an easy way to steal someone's API keys if api_url is configurable by mistake